README.md

```diff
@@ -51,13 +52,13 @@ Two types of release streams are available:
 2. Rolling Stable Releases:
    - Use Rolling Stable releases for early access to new features/new hardware. Rolling updates include a mix of feature changes and bug/security fixes. Risks of new bugs and regressions are higher for rolling stable than for production. Customers must install the next release for any updates, including bug fixes. Major and minor OS version support are locked.
 
+## Active LTS/Production releases:
 
-## Active Production release:
-
-| Release | Branch| Tag|
-|---    |---    |--- |
-| 2328.48:update released 20231229 | backport |[I915_23WW51.5_682.48_23.6.42_230425.56](https://github.com/intel-gpu/intel-gpu-i915-backports/blob/backport/RELEASE_2335_23.6/README.md)|
-| | redhat | [RHEL89_23WW51.5_682.48_23.6.42_230425.55](https://github.com/intel-gpu/intel-gpu-i915-backports/blob/redhat/RELEASE_2335_23.6/README.md)|
+| Release | Type| Branch| Tag| Status|
+|---    |---    |---    |---    |---    |
+| 2350.29: initial released 20240131 | LTS | backport |[I915_24WW05.5_803.29_23.10.32_231129.32](https://github.com/intel-gpu/intel-gpu-i915-backports/blob/backport/RELEASE_2405_23.10/README.md)| Active|
+| 2328.48:update released 20231229 | Production | backport |[I915_23WW51.5_682.48_23.6.42_230425.56](https://github.com/intel-gpu/intel-gpu-i915-backports/blob/backport/RELEASE_2335_23.6/README.md)| Obsolete |
+| | Production | redhat | [RHEL89_23WW51.5_682.48_23.6.42_230425.55](https://github.com/intel-gpu/intel-gpu-i915-backports/blob/redhat/RELEASE_2335_23.6/README.md)| Obsolete |
 
```

Comment:
Branch does not look correct, pls correct that before merging.
23.6 --> backport/RELEASE_XYZ_23.6
           -> readhat/RELEASE_XYZ_23.6
similarly for 23.10

Reply:
done